pages/settings/privacy/index.vue

```diff
@@ -0,0 +1,52 @@
+<script setup lang="ts">
+import Glean from '@mozilla/glean/web'
+import { mastodonAccountHandle, mastodonAccountId } from '~/telemetry/generated/identifiers'
+import { userAgent } from '~~/telemetry/generated/identifiers'
+
+const user = currentUser.value
+
+const { t } = useI18n()
+
+useHydratedHead({
+  title: () => `${t('settings.privacy.label')} | ${t('nav.settings')}`,
+})
+const userSettings = useUserSettings()
+
+function toggleOptOut() {
+  const allowGlean = togglePreferences('allowGlean')
```

Comment:
This is where I'm using that explicit return from `togglePreferences`. For some reason calling `getPreferences` after doing the toggle wasn't returning the correct value and I didn't have the time or mental space to explore why

Reply:
Just wondering, did you try calling `getPreferences` before doing the toggle?